tests/ConstructAndDestroyComponentTests.js

```diff
@@ -0,0 +1,72 @@
+"use strict";
+
+var fluid = require("infusion"),
+    gpii = fluid.registerNamespace("gpii"),
+    kettle = fluid.registerNamespace("kettle"),
+    path = require("path"),
+    configPath = path.resolve(__dirname, "configs");
+
+require("kettle");
+require("../src/Nexus.js");
+require("../src/test/NexusTestUtils.js");
+
+kettle.loadTestingSupport();
+
+fluid.registerNamespace("gpii.tests.nexus.constructComponent");
+
+gpii.tests.nexus.constructComponent.componentOptions = {
+    type: "fluid.modelComponent",
+    model: {
+        someModelPath: 2
+    }
+};
+
+gpii.tests.nexus.constructComponent.testDefs = [
+    {
+        name: "Construct and Destroy Component",
+        gradeNames: "gpii.test.nexus.testCaseHolder",
+        expect: 6,
+        config: {
+            configName: "gpii.tests.nexus.config",
+            configPath: configPath
+        },
+        testComponentPath: "nexusConstructComponentTestComponent",
```

Comment:
Try something a bit more exotic here to test out the EL parsing that you will put in in the core impl - e.g. an array or a path containing escaped periods (preferably both!)